indexing-service/src/main/java/org/apache/druid/indexing/overlord/TaskLockbox.java

```diff
@@ -710,8 +710,8 @@ public TaskLock apply(TaskLockPosse taskLockPosse)
                             // Do not proceed if the lock is revoked
                             return;
                           } else if (taskLockPosse.getTaskLock().getPriority() == null
-                                     || taskLockPosse.getTaskLock().getPriority() <= minTaskPriority.get(datasource)) {
-                            // Do not proceed if the lock has a priority less than or equal to the minimum
+                                     || taskLockPosse.getTaskLock().getPriority() < minTaskPriority.get(datasource)) {
```

Comment:
Why is this changed back?

Reply:
I had missed changing it the first time. 

The behaviour after this change is that intervals of tasks with equal priority will be considered locked. That is the required behaviour, right?

Reply:
LGTM